e3sm_to_cmip/__main__.py

```diff
@@ -717,9 +720,40 @@ def _run_info_mode(self):  # noqa: C901
                             break
                     if not has_vars:
                         continue
-                    messages.append(msg)
 
-        if self.output_path is not None:
+                    # We test here against the input "freq", because atmos mon data satisfies BOTH
+                    # CMIP6_day.json AND CMIP6_mon.json, but we only want the latter in the "hand_msg" output.
+                    # The variables-check below is insufficient as vars "hass" and "rlut" have multiple freqs.
```

Comment:
Not sure if these comments are within 80 characters per line

Reply:
In total, no.  But does one begin measuring at the "#" or at column 1?  (Never knew very long comment lines to be an issue.)